src/scrollbox.js

```diff
@@ -490,47 +571,45 @@ export class Scrollbox extends PIXI.Container {
      * @private
      */
     scrollbarDown(e) {
+        const bWidth = this.scrollBarWidthOverride ? this.scrollBarWidthOverride : this.boxWidth
```

Comment:
bWidth - boxWidth I guess - maybe just make that explicit

Reply:
Updated 👍